docs/shared/tools-workspace-schematics.md

```diff
@@ -1 +1,284 @@
 # Using Workspace Schematics
+
+Workspace schematics provide a way to automate many tasks you regularly perform as part of your development workflow. Whether its scaffolding out components, features, or ensuring libraries are generated and structured in a certain way, schematics help you standardize these tasks in a consistent, and predictable way. Nx provides tooling around creating, and running custom schematics from within your workspace. This guide shows you how to create, run, and customize workspace schematics within your Nx workspace.
```

Comment:
Type: _"Whether **it is** scaffolding out components,..."_

I would maybe split this sentence as follows:

Maybe avoid repetition of "way" ?
_"Whether ....structured in a certain way, schematics help you standardize these tasks in a consistent, and predictable manner."_